src/auth/types.ts

```diff
@@ -90,7 +93,8 @@ export interface IncomingAuthType {
   }
   disableLocalStrategy?: true
   strategies?: {
-    strategy: Strategy
+    name: string
```

Comment:
We'd like to see this being done in a non-breaking change sort of way.
First, name can be optional.

Reply:
Yes, I thought about that, but since this is a so early stage feature, I wonder if anybody is even using it yet, since there isn't yet documentation around it. Either way, I am fine making this flexible.